docs/root/start/install.rst

```diff
@@ -171,6 +168,29 @@ The following commands will pull and show the Envoy version of current images.
          To use the ``nightly`` version from `Get Envoy <https://www.getenvoy.io/>`__
          replace the word ``stable`` with ``nightly`` in the above commands.
 
+   .. tab:: Windows (image)
+
+      You can run Envoy on a Windows host using the official Windows Docker image.
+
+      .. warning::
```

Comment:
What is the value of this warning? Windows is going GA, next week and the code is already complete. I would remove it

Reply:
2 things:

- afaict these images are experimental - and should be treated as such
- in an open source project documenting and/or distributing binary blobs should come with a warning

this is "bringing parity" - i would add similar warnings to any other code/images etc

Reply:
>  Why do you think they are experimental? 

at least afaict they have reduced functionality and are less performant - if these images are genuinely production-ready we can remove this bit - but im already seeing "it doesnt work" tickets, so my concern is both to be explicit about the fact that they are not widely tested and to add a "caveat usor"

> If this is the case shouldn't all images have this warning then?

the other images *are* open source - they dont need it - there is no code in them for which the source is not available - and the recipes are published

Reply:
... and the user can compile the entire  stack for themselves should they wish

Reply:
s/distributing binary blobs/distributing closed-source binary blobs/

Reply:
That is a good opportunity to clarify the ambiguities.

> at least afaict they have reduced functionality and are less performant

re feature parity: There are some features that are missing which we explicitly document but neither of the missing features is deemed high priority from customers.

re performance: This is not actually true. We have comparable median TCP termination performance on Windows and somewhat reduced p95 performance. There were some issues in the alpha release regarding performance which we addressed.

re tickets: I have not seen a production bug yet but I have seen questions regarding usability. This is an opportunity for us to improve the documentation. We already have teams within msft that are adopting Envoy on Windows in production.

Reply:
Regarding the disclaimer for the open source code I will defer to @wrowe to chime in here

Reply:
I have seen this and I have this on my radar. This is a question for now.

Reply:
also worth pointing out the *compressed* size windows image is 2.3GB (ie the compressed size of the not open-source binary blobs), whereas the linux equivalent is 50MB

even if this is "production-ready" its not afaict suitable for production use - esp not in the situation that Envoy is most typically deployed in - ie as a sidecar

Reply:
@envoyproxy/windows-dev 

Would it be possible to get more context about closed source binaries included in the windows image and where they come from?

@wrowe and Windows Envoy maintainers: what would be your preferred wording for this section?  Worth adding some notes about size and plans to improve on that?

Reply:
The Windows Server Core base image usually comes pre-cached on a lot of cloud provided images etc., so its size is an issue but the impact can be minimized. Definitely already being used in production in other products (cloudfoundry, etc.). It should consist of 2 layers, the base that is tied to the host version, and another layer with Windows updates that gets updated periodically. B/c of the host/container image compatibility requirements now, the large base layer won't change (much/ ever) I believe in the lifecycle of a Windows version.

Reply:
Also the base container image for Windows is not being distributed by Envoy/DockerHub per-se, the Windows proprietary/licensed layers do not get hosted on DockerHub, they are foreign layers that still come from Microsoft when you pull the image (the image manifest should show this) that have their own license deal. I think the Envoy project is only on the hook for the layers that we build on top of the Windows base.

Reply:
> Worth adding some notes about size and plans to improve on that?

The plans are related to reducing the size of the base image. I do not think that Envoy docs is the best channels to communicate such changes though.

Reply:
Let's discuss in our meeting tomorrow morning, but in general my view is:
1) Remove the experimental warning. We deemed it GA and users understand when it was GA, can open bugs, etc.
2) Remove the not open source warning (Envoy itself is Apache 2, and any base image is owned by Microsoft, and anyone using Windows understands what they are dealing with). If there are legal concerns here we can run them by the CNCF.
3) Optionally add a warning about size and link to the tracking issue, though I tend to agree with @wrowe that anyone that downloads the image knows it's big. There may be value though in telling people the tracking issue.